pyscriptjs/Makefile

```diff
@@ -10,8 +10,25 @@ CONDA_EXE := conda
 CONDA_ENV ?= ./env
 env := $(CONDA_ENV)
 conda_run := $(CONDA_EXE) run -p $(env)
+GOOD_NODE_VER := 14
+GOOD_NPM_VER := 6
+NODE_VER    := $(shell node -v | cut -d. -f1 | sed 's/^v\(.*\)/\1/')
+NPM_VER     := $(shell npm -v | cut -d. -f1)
+
+GOOD_NODE   := $(shell if [ $(NODE_VER) -ge $(GOOD_NODE_VER) ]; then echo true; else echo false; fi)
+GOOD_NPM    := $(shell if [ $(NPM_VER) -ge $(GOOD_NPM_VER) ]; then echo true; else echo false; fi)
```

Comment:
Is there an advantage of using `-ge`? Or is it equivalent to `>=`? I feel like the latter is better.

Reply:
I don't think there's an advantage. Just always used (and seen) this way... I think double parenthesis can be used as well for performing numeric comparison but may be wrong here...